client/post-editor/editor-featured-image/index.jsx

```diff
@@ -10,12 +10,12 @@ import { bindActionCreators } from 'redux';
  */
 const MediaLibrarySelectedData = require( 'components/data/media-library-selected-data' ),
 	EditorMediaModal = require( 'post-editor/media-modal' ),
-	EditorDrawerWell = require( 'post-editor/editor-drawer-well' ),
 	PostActions = require( 'lib/posts/actions' ),
 	PostUtils = require( 'lib/posts/utils' ),
 	stats = require( 'lib/posts/stats' ),
-	AccordionSection = require( 'components/accordion/section' ),
 	EditorFeaturedImagePreviewContainer = require( './preview-container' );
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
```

Comment:
>Maybe tidy the rest of the imports in a follow up?

Missed the last few words of your comment and went ahead with changes here 56582d03829936b2297f35776fb7a41d8ecb9e05.